target/linux/ramips/dts/mt7621_xiaomi_mir3p.dts

```diff
@@ -171,9 +171,29 @@
 	};
 };
 
-&ethernet {
+&gmac0 {
 	mtd-mac-address = <&factory 0xe000>;
-	mediatek,portmap = "llllw";
+};
+
+&switch0 {
+	ports {
+		port@1 {
+			status = "okay";
+			label = "lan3";
+		};
+		port@2 {
+			status = "okay";
+			label = "lan2";
+		};
+		port@3 {
+			status = "okay";
+			label = "lan1";
+		};
+		port@4 {
+			status = "okay";
+			label = "wan";
```

Comment:
This device needs the same setup as mir3g (v1), so wan 0xe000 and gmac 0xe006, as well as label-mac-device = gmac added.